src/main/java/lucas/duarte/jazz/controller/PartidaController.java

```diff
@@ -20,34 +24,64 @@
 public class PartidaController {
 	@Autowired
 	private PartidaService partidaServ;
+	@Autowired
+	private ExceptionController exceptController;
 
-	// Get all partidas
 	@RequestMapping(value = "/partidas/", method = RequestMethod.GET)
-	public ResponseEntity<List<Partida>> listAllpartidas() {
-		return partidaServ.getAllPartidas();
+	public ResponseEntity<?> listAllpartidas() {
+		List<Partida> partidas = partidaServ.getAllPartidas();
+
+		if (partidas.isEmpty()) {
```

Comment:
essa validação pode estar na service, e retorna o Response de lá caso não exista partidas.

Reply:
Nao, como disse no grupo do wpp, o Response Entity tem que ser tratato no controller somente, e não na camada de regra de negocio, isso está nas boas práticas do Clean Architecture, pois assim diminuimos o acoplamento do controller com o service, vamos refatorar todas.

Reply:
na verdade a validação do is empty é equivalente a if null no controller, por tanto não tem diferença nesse caso, apenas retorna um Json vazio